donations/views.py

```diff
@@ -185,6 +185,10 @@ def donation_session_stripe(request):
               cancel_url=return_url_cancel,
             )
             return JsonResponse({"session_id":session.id})
+        else:
+            return JsonResponse({'errors': form.errors})
```

Comment:
In the case that the form is invalid are you going to continue returning 200, or also 400?

Reply:
200, like in 'donation_session_paypal'